google/cloud/pubsublite/publisher_connection.cc

```diff
@@ -147,10 +148,18 @@ StatusOr<std::unique_ptr<PublisherConnection>> MakePublisherConnection(
   auto publisher_service_stub = CreateDefaultPublisherServiceStub(cq, opts);
   auto batching_options = MakeBatchingOptions(opts);
 
+  bool enable_idempotence = true;
```

Comment:
In all other libraries we do something like (paraphrasing):

```
std::unique_ptr<PublisherConnection> MakePublisherConnection(
    Topic topic, Options opts) {
  opts = ApplyDefaults(std::move(opts));
```

for example:

https://github.com/googleapis/google-cloud-cpp/blob/e11ad2c3a59eeff9ddb9f435fe71b4f89672a9a4/google/cloud/pubsublite/admin_connection.cc#L167-L173

and then we do not need to worry about what options are set.  Please consider *setting* this option in `opts` so a future refactoring is easier: